packages/grafana-data/src/transformations/transformers/seriesToColumns.test.ts

```diff
@@ -148,7 +146,102 @@ describe('SeriesToColumns Transformer', () => {
       (received) => {
         const data = received[0];
         const filtered = data[0];
-        expect(filtered.fields).toMatchInlineSnapshot(`Array []`);
```

Comment:
This test took me a long time to realize why it changed.  Scanning the code, joining the code by "temperature" should absolutely lead to the results below.

BUT... previously when checking for display name, the the fields would render to `even temperature` and `odd temperature` -- so they would not join.  :exploding_head: -- works for time because we don't do any magic